virt/virtual_machines/virt-accessing-vm-consoles.adoc

```diff
@@ -25,6 +25,8 @@ include::modules/virt-vm-rdp-console-web.adoc[leveloffset=+2]
 
 include::modules/virt-switching-displays.adoc[leveloffset=+2]
 
+include::modules/virt-copying-the-ssh-command.adoc[leveloffset=+2]
+
 [role="_additional-resources"]
```

Comment:
This **Additional resources** section belongs to the previous module. You will need to move it up, after the `include::modules/virt-switching-displays.adoc[leveloffset=+2]` line.